docker-compose.yml

```diff
@@ -94,10 +94,12 @@ services:
             context: .
             dockerfile: ./docker/rabbit-mq/Dockerfile
         hostname: rabbit-mq
-        ports:
-            - "8083:15672"
```

Comment:
It has already been removed. However, the overview dashboard isn't yet a replacement for the whole management interface. A dashboard for queues and possibly exchanges still needs to be added.